test/models/list_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class ListTest < ActiveSupport::TestCase
+  fixtures :lists
+
```

Comment:
@vlasiak, I misremembered to mention that as there is an association between list and items it would be great also to add a test asserting that a list has items.